nemo_skills/evaluation/evaluator.py

```diff
@@ -432,7 +432,9 @@ def eval_lean4(cfg):
     'mt-bench': eval_mtbench,
     'answer_judgement': dummy_eval,
     'lean4': eval_lean4,
-    'mmlu-pro': eval_mmlu_pro,
+    'mmlu-pro': eval_mcq,
```

Comment:
Can we instead have a single eval type called 'mcq' and use it in directly in all relevant benchmarks?